### ISSUE TITLE
Code of Conduct

### ISSUE THREAD

**Issue by hendrikebbers** [eclipse-ee4j/ee4j#70]
Eclipse provide a Code of Conduct here: https://www.eclipse.org/org/documents/Community_Code_of_Conduct.php

From my point of view it would make sense to add a CODE_OF_CONDUCT to all repos that contains the Eclipse CoC.

If this is a good idea I can provide the PRs ;)

**Comment by anthonyvdotbe** [comment]
Since the Eclipse CoC is versioned, and there's already a `CONTRIBUTING.md` file in each repo, I think it would make more sense to add a reference to the Eclipse CoC in the `CONTRIBUTING.md` file.

**Comment by hendrikebbers** [comment]
Fine with that :) I do not know if there are some tool that scan for CODE_OF_CONDUCT files or some other benefits by doing it in the "standard GitHub way".
<img width="1433" alt="Bildschirmfoto 2020-10-04 um 16 10 07" src="https://user-images.githubusercontent.com/9443847/95017896-2dcb7b00-065c-11eb-929c-1fcdb52c9c56.png">
But even if there would be a specific CODE_OF_CONDUCT file we should link directly to the Eclipse CoC instead of adding a copy

**Comment by pzygielo** [comment]
With https://github.com/eclipse-ee4j/.github/ in place - this one perhaps can be closed.

**Comment by ivargrimstad** [comment]
Yes, thanks for pointing it out. Had forgotten about this issue :)
